src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -308,27 +311,38 @@ class MaybeDependency:
 
 
 class DependencyResolver:
```

Comment:
Please add the reasoning to separate the notebook resolver from the import resolvers to the docstring

Reply:
Done